autospec/config.py

```diff
@@ -554,7 +559,17 @@ def parse_existing_spec(path, name):
 
 def parse_config_versions(path):
     """Parse the versions configuration file."""
-    return set(read_conf_file(os.path.join(path, "versions")))
+    versions = OrderedDict()
+    for line in read_conf_file(os.path.join(path, "versions")):
+        fields = line.split()
+        version = fields.pop(0)
+        if len(fields):
+            url = fields.pop(0)
+        else:
+            url = ""
+        versions[version] = url
```

Comment:
Hrm do we want to have some oops detection for somebody putting in multiple urls for the same version by accident?

Reply:
Currently any subsequent fields will be ignored when reading, and omitted when writing. Do you think we should report their existence? Could be done.

Reply:
Yea, just a report warning so folks can see something.